command/rootchain/helper/metadata.go

```diff
@@ -88,3 +90,28 @@ func ReadRootchainIP() (string, error) {
 
 	return fmt.Sprintf("http://%s:%s", ports[0].HostIP, ports[0].HostPort), nil
 }
+
+// JSONRPC returns a new jsonrpc client on the given endpoint
+func JSONRPC(endpoint string) (*jsonrpc.Client, error) {
+	clt, err := jsonrpc.NewClient(endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("could not open client. Error: %w", err)
+	}
+
+	return clt, nil
+}
+
+// GetGasPriceOnRoot returns gas price
+func GetGasPriceOnRoot(jsonrpcEndpoint string) (uint64, error) {
+	client, err := JSONRPC(jsonrpcEndpoint)
+	if err != nil {
+		return 0, err
+	}
+
+	gasPrice, err := client.Eth().GasPrice()
+	if err != nil {
+		return 0, fmt.Errorf("could not get gas price on root. Error: %w", err)
+	}
+
+	return gasPrice, nil
+}
```

Comment:
I would do this differently:
- TxRelayer is anyways instantiating `jsonrpc.Client`, so why not expose it, by introducing new function `GetJSONRPCClient`
- then just call `GetJSONRPCClient` and query whatever JSON RPC endpoint we wish.

These helper functions are almost useless.

Reply:
I don't think they are useless, since 3 commands are calling the same code. I can expose the client on the `txRelayer` and use it on this `GetGasPriceOnRoot`.

Reply:
Or better yet, I'll expose a `GetGasPrice` on the relayer itself.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1408/commits/4e7861ee9a79a63c65228eef35351689ca51e5f9